### PR TITLE
Update dependencies example for mvn to have 'provided' scope

### PR DESCRIPTION
**Pull request by sbaum1994**
Not sure if I worded that bullet point right.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,31 +22,40 @@ additional installation and setup information for using the library with various
...
 
+<!-- Provided Dependencies -->
 <dependency>
-      <groupId>io.grpc</groupId>
-      <artifactId>grpc-netty</artifactId>
-      <version>1.4.0</version>
+    <groupId>org.slf4j</groupId>
+    <artifactId>slf4j-simple</artifactId>
+    <version>1.7.25</version>
+    <scope>provided</scope>
```
Comment (MatthewDolan) [inline README.md]: I don't think we want `<scope>provided</scope>` here. We declare our dependencies as provided, but we expect someone who is including the our dependencies to do the providing. I might be wrong though.

**Comment by jmacd** [comment]
lgtm

File: README.md (hunk 1 above)
Comment (sbaum1994) [reply README.md]: @MatthewDolan ping

Comment (MatthewDolan) [reply README.md]: Maybe, I think it just isn't very clear. The expectation is that you do have these dependencies bound in your pom.xml. Can we chat IRL?

**Pull request merged**